include/mata/nfa.hh

```diff
@@ -226,9 +228,14 @@ struct TransSymbolStates {
     inline bool operator==(const TransSymbolStates& rhs) const { return symbol == rhs.symbol; }
 };
 
+/// List of transitions from a certain state. Each element holds transitions with a certain symbol.
 using TransitionList = Mata::Util::OrdVector<TransSymbolStates>;
+/// Transition relation for an NFA. Each index 'i' to the vector represents a state 'i' in the automaton.
 using TransitionRelation = std::vector<TransitionList>;
 
+/**
+ * A struct representing an NFA.
```

Comment:
You can leave this comment here, but for the next time, I would avoid comments like this. It does not explain anything, just comment obvious using plain text. Do you agree @tfiedor?

Reply:
I agree. Nevertheless, if you generate documentation with Doxygen, I think one would appreciate a comment to be shown for each structure or class, though. Otherwise, I feel like there is something missing from the documentation. At this current state for `struct Nfa` when you are reading the header file instead of the generated documentation, however, the comment will not tell you much new.

Reply:
> Just few suggestion, what should be there in documentation: limitations, example of usage, what NFA means (yeah for us it is obvious), caveats, possible exceptions, etc.

Which we should be thinking about adding to many places in the project, I think.